static.json

```diff
@@ -2,6 +2,7 @@
     "root": "build/",
     "clean_urls": true,
     "absolute_redirect": false,
+    "error_page": "build/404.html",
```

Comment:
Waiting for the deployment to see if the path is the right one or if I should skip the `build/` part